From 7976e335770daa36680e8bc16e6057bbe7910203 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sun, 28 Mar 2021 11:02:25 +0200
Subject: [PATCH 1/1] sandbox: document parameters of os_realloc()

Avoid 'make htmldocs' build warnings:

    ./include/os.h:139: warning:
    Function parameter or member 'ptr' not described in 'os_realloc'
    ./include/os.h:139: warning:
    Function parameter or member 'length' not described in 'os_realloc'

Fixes: 14e46dfb176b ("sandbox: Add os_realloc()")
Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 include/os.h | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/include/os.h b/include/os.h
index 77d8bd89d0..bd1096eb8b 100644
--- a/include/os.h
+++ b/include/os.h
@@ -134,7 +134,9 @@ void os_free(void *ptr);
  * This follows the semantics of realloc(), so can perform an os_malloc() or
  * os_free() depending on @ptr and @length.
  *
- * Return:	Pointer to reallocated memory or NULL if @length is 0
+ * @ptr:	pointer to previously allocated memory of NULL
+ * @length:	number of bytes to allocate
+ * Return:	pointer to reallocated memory or NULL if @length is 0
  */
 void *os_realloc(void *ptr, size_t length);
 
-- 
2.30.2

